.github/settings.yml

```diff
@@ -143,7 +143,7 @@ collaborators:
   - username: 92nqb
     permission: push
 
-  # l10n zh approvers
+  # l10n zh-ch approvers
```

Comment:
`zh-cn` would be correct 😅
( https://glossary.cncf.io/zh-cn/ )

```suggestion
  # l10n zh-cn approvers
```

Reply:
Oh! Thanks. Updated.